### PR TITLE
Implemented update on create customer and get transaction

### PR DESCRIPTION
**Pull request by azeezlala**

### CODE REVIEW

File: model/model.go [hunk 1]
```diff
@@ -337,4 +348,17 @@ type (
...
+	Page struct {
+		Page int64 `json:"page"`
+	}
+
+	// TransactionResponse object
+	TransactionResponse struct {
+		Item struct {
+			Transactions []*Transaction `json:"transactions"`
+		} `json:"item"`
+		Page Page `json:"page"`
```
Comment (odetolakehinde) [inline model/model.go:362]: why does this return only page number?

Comment (azeezlala) [reply model/model.go:362]: Boss the Page na struct

Comment (odetolakehinde) [reply model/model.go:362]: the reference struct is just an int na.

File: model/model.go [hunk 2]
```diff
@@ -337,4 +348,17 @@ type (
 		TotalBalance float64                    `json:"total_balance"`
 		Detail       []*CustomerBalanceResponse `json:"detail"`
 	}
+
+	// Page object
+	Page struct {
+		Page int64 `json:"page"`
+	}
```
Comment (odetolakehinde) [inline model/model.go:355]: @azeezlala see this na.

File: model/model.go [hunk 3]
```diff
@@ -325,10 +336,10 @@ type (
 
 	// CustomerBalanceResponse object
 	CustomerBalanceResponse struct {
-		YieldOfferingID uuid.UUID `json:"yield_offering_id"`
-		Name            string    `json:"name"`
-		Currency        string    `json:"currency"`
-		Amount          float64   `json:"balance"`
+		YieldOfferingID []uuid.UUID `json:"yield_offering_id"`
```
Comment (wharleyinc) [inline model/model.go:339]: `YieldOfferingIDs`

File: model/model.go [hunk 4]
```diff
@@ -32,21 +32,21 @@ const (
...
-		Email           string `json:"email"`
-		Reference       string `json:"reference"`
-		MobileNumber    string `json:"mobile_number"`
-		YieldOfferingID string `json:"yield_offering_id"`
+		CustomerID      string      `json:"customer_id"`
+		Name            string      `json:"name"`
+		Email           string      `json:"email"`
+		Reference       string      `json:"reference"`
+		MobileNumber    string      `json:"mobile_number"`
+		YieldOfferingID []uuid.UUID `json:"yield_offering_id"`
```
Comment (wharleyinc) [inline model/model.go:49]: YieldOfferingIDs

File: model/model.go [hunk 5]
```diff
@@ -32,21 +32,21 @@ const (
...
-		Name            string `json:"name"`
-		Email           string `json:"email"`
-		Reference       string `json:"reference"`
-		MobileNumber    string `json:"mobile_number"`
-		YieldOfferingID string `json:"yield_offering_id"`
+		Name            string      `json:"name"`
+		Email           string      `json:"email"`
+		Reference       string      `json:"reference"`
+		MobileNumber    string      `json:"mobile_number"`
+		YieldOfferingID []uuid.UUID `json:"yield_offering_id"`
```
Comment (wharleyinc) [inline model/model.go:39]: 👀

File: model/model.go [hunk 6]
```diff
@@ -38,21 +38,21 @@ const (
...
-		Name            string `json:"name"`
-		Email           string `json:"email"`
-		Reference       string `json:"reference"`
-		MobileNumber    string `json:"mobile_number"`
-		YieldOfferingID string `json:"yield_offering_id"`
+		Name             string      `json:"name"`
+		Email            string      `json:"email"`
+		Reference        string      `json:"reference"`
+		MobileNumber     string      `json:"mobile_number"`
+		YieldOfferingIDs []uuid.UUID `json:"yield_offering_id"`
```
Comment (wharleyinc) [inline model/model.go:45]: the json field

File: model/model.go (hunk 3 above)
Comment (wharleyinc) [reply model/model.go:356]: @azeezlala

**Review by odetolakehinde — DISMISSED** [review]

**Review by wharleyinc — APPROVED** [review]

**Pull request merged**